src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -79,7 +79,7 @@ public function getEnv(string $prefix, string $name, \Closure $getEnv)
             }
 
             if (!isset($array[$key]) && !\array_key_exists($key, $array)) {
-                throw new EnvNotFoundException(sprintf('Key "%s" not found in %s (resolved from "%s").', $key, json_encode($array), $next));
+                throw new EnvNotFoundException(sprintf('Key "%s" not found in "%s" (resolved from "%s").', $key, json_encode($array), $next));
```

Comment:
please revert his one, that's a false positive

Reply:
@nicolas-grekas fabbot.io told me so, but okay

Reply:
that's exactly what Nicolas said: it was a false positive of fabbot.io. The tool is not 100% perfect.

Reply:
ok, revert is done